README.md

```diff
@@ -82,163 +82,41 @@ For that reason, I didn't do too much here. That said, there is probably a lot
 that can/should be disabled in Rails that is there by default for "free" if you
 wanted this to be a closer comparison.  Pull requests welcome if you care!
 
+## Updates!  Round two of benchmarks (January 2015)
```

Comment:
@josevalim this is the section I was attempting to address the added functionality of Phoenix in.